src/Files.App/Files.App.csproj

```diff
@@ -45,7 +45,7 @@
         <Content Include="7z64.dll">
             <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
         </Content>
-        <Content Include="..\..\NOTICE.md">
+        <Content Include="..\..\.github\NOTICE.md">
```

Comment:
Does the about page need to be updated to use this path?

Reply:
Thank you for having pointed it out. Fixed.